homework-2/services/userService.ts

```diff
@@ -0,0 +1,52 @@
+import { User } from "../interfaces/User";
+import { v4 as uuidv4 } from 'uuid';
+import { initialUsersList } from "../config/constants";
+
+export default class UserService {
```

Comment:
Using static class is strange desision

Reply:
I've replace static class with common class + export default